.changeset/thin-bags-repair.md

```diff
@@ -0,0 +1,9 @@
+---
+"@comet/admin": minor
+---
+
+Add `useDataGridExcelExport` hook for exporting data from a `DataGrid` in an excel file `useDataGridExcelExport` returns an `exportApi` encompassing
+
+-   `exportGrid()` to generate and export the excel file
+-   `loading`
+-   `error`
```

Comment:
This sounds more like two sentences where the list below is part of the second one. 
Also it's not really clear, what `loading` and `error` are/do. 
Would this make more sense?

```suggestion
Add the `useDataGridExcelExport` hook for exporting data from a `DataGrid` to an excel file 

The hook returns an `exportApi` encompassing: 

-   `exportGrid`: a function to generate and export the excel file
-   `loading`: a boolean indicating if the export is in progress
-   `error`: an error message when the export has failed
```